src/python/pants/core/util_rules/bootstrap.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I'm not sure `bootstrap` is the right name for this namespace; perhaps `search_paths` would be a better fit? We certainly expect more rules relating to search path manipulation (see e.g. #14492).

Reply:
Sure!